subprojects/packagefiles/asio/meson.build

```diff
@@ -0,0 +1,22 @@
+project('asio',
+    'cpp',
+    version : '1.24.0',
+    license : 'BSL-1.0',
+)
+
+asio_deps = []
+
+# For windows builds, ASIO depends on ws2_32
+if target_machine.system() == 'windows'
+    compiler = meson.get_compiler('cpp')
+    ws2_32_dep = compiler.find_library('ws2_32', required: true)
```

Comment:
You don't need `required: true`. It is the default.

Reply:
Ah classic copy-pasta, thx!